docs/core/traced.md

```diff
@@ -0,0 +1,119 @@
+# traced Decorator
+
+The `traced()` decorator enables automatic tracing of function calls, inputs, and outputs using OpenTelemetry. It is designed to help you monitor, debug, and audit your code by capturing detailed information about function executions, including arguments, return values, and exceptions. Traces are exported to the configured OpenTelemetry backend.

```

Comment:
include details about the `Tracing` tab found in the UI

Reply:
Added more info

```
You can view the traces of an Orchestrator job by going to the Jobs page, click a job, a side panel will open, and they will be available under the `Trace` tab. These can also be seen in UiPath Maestro when your agent is part of a larger process orchestration.
```